### ISSUE TITLE
Error when using prismic in nuxt dev tools

### ISSUE THREAD

**Issue by Applelo** [nuxt-modules/prismic#212]
### Versions

- nuxt: 3.11.2
- @nuxtjs/prismic: 3.2.0
- node: 20.11.1

### Reproduction

https://stackblitz.com/edit/github-zwe7ab?file=slicemachine.config.json

### Steps to reproduce

Open the devtools and select the prismic tab throw the following error. To throw the error, the slicemachine.config.json need to be here.

```
 ERROR  [nuxt-devtools] RPC error on executing "prismic-slicemachine-rpc:getSlicemachineConfig":        9:41:43 AM
Module "file:///Users/lboubault/Sites/julbo/julbo/slicemachine.config.json" needs an import attribute of type "json"
```
### What is Expected?

Not to throw an error and read the slicemachine.config.json file.

### What is actually happening?

The prismic tab in nuxt dev tools is not usable with user with slicemachine.config.json file.

**Comment by lihbr** [comment]
Hey there, thanks! I noticed this was happening with the published artifact yesterday, it'll be fixed today ☺️

**Comment by lihbr** [comment]
Released in [3.2.1](https://github.com/nuxt-modules/prismic/blob/master/CHANGELOG.md#321-2024-04-09) 🎉

Should be working as-in: https://stackblitz.com/edit/github-zwe7ab-npogug?file=package.json

Thanks again for trying the new release so quickly and reporting the issue. The devtools integration is very minimal for now but we wanted to set up the foundation for it first before iterating~
